sources/es/domentranslations.py

```diff
@@ -14,7 +14,7 @@ class CclawTranslations(Crawler):
 
     def initialize(self) -> None:
         self.cleaner.blacklist_patterns.update(['CONTENIDO | SIGUIENTE'])
-    # end def
+
```

Comment:
changing sources will cause all users to get a force update. imagine you are running the app for a quick use, and having to wait for 300+ crawlers to fnish downloading. we should not modify all sources for just removing comments